<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MorionQTApp</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 178, 102, 255), stop:0.55 rgba(235, 148, 61, 255), stop:0.98 rgba(0, 0, 0, 255), stop:1 rgba(0, 0, 0, 0));</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>120</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 222, 144);</string>
    </property>
    <property name="text">
     <string>Загрузить Базу</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../../../Desktop/тестовое задание/medium_f0d7b9975fe120cf5d14799c11606a99.png</normaloff>../../../Desktop/тестовое задание/medium_f0d7b9975fe120cf5d14799c11606a99.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>581</width>
      <height>421</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTableView" name="tableView">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(207, 207, 207);</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="BlockB">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 222, 144);</string>
         </property>
         <property name="text">
          <string>Блоки</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../Desktop/тестовое задание/free-icon-cloud-database-2637294.png</normaloff>../../../Desktop/тестовое задание/free-icon-cloud-database-2637294.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="PortB">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 222, 144);</string>
         </property>
         <property name="text">
          <string>Платы</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../Desktop/тестовое задание/free-icon-cloud-database-2637294.png</normaloff>../../../Desktop/тестовое задание/free-icon-cloud-database-2637294.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="BoardB">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 222, 144);</string>
         </property>
         <property name="text">
          <string>Порты</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../../../Desktop/тестовое задание/free-icon-cloud-database-2637294.png</normaloff>../../../Desktop/тестовое задание/free-icon-cloud-database-2637294.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>18</width>
           <height>18</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>69</y>
      <width>201</width>
      <height>53</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QLabel" name="ConnectionStatus">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 255, 255);</string>
       </property>
       <property name="text">
        <string>     Connection status:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QPushButton" name="begin">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(204, 204, 204);</string>
       </property>
       <property name="text">
        <string>Начать сеанс</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>../../../Desktop/тестовое задание/free-icon-plug-and-play-6878161.png</normaloff>../../../Desktop/тестовое задание/free-icon-plug-and-play-6878161.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>18</width>
         <height>18</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
